### PR TITLE
Updated image namespace for Patroni V10 to bcgov (from openshift)

### PR DESCRIPTION
**Pull request by arcshiftsolutions**

### CODE REVIEW

File: openshift/sso72-x509.yaml [hunk 1]
```diff
@@ -242,7 +242,7 @@ objects:
         - "rh-sso"
         from:
           kind: ImageStreamTag
-          name: "${NAME}:${IMAGE_TAG}"
+          name: "${NAME}:${VERSION}"
```
Comment (cvarjao) [inline openshift/sso72-x509.yaml]: `VERSION` is not intended to be used as tag. It is only for metadata/label of the version of RHSSO. Please undo this change.

Comment (arcshiftsolutions) [reply openshift/sso72-x509.yaml]: Will do Clecio. This change was made as I was still trying to use the OCP-SSO documentation to deploy; should have deleted this PR prior, apologies.
